spec/rails_helper.rb

```diff
@@ -21,7 +21,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
```

Comment:
why sort ?
don't you have a rubocop message on this line ... asking for something more similar to the previous one:
```ruby
Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
```
see: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/FilePath

Reply:
It was in the correction. I hesitated to change. The initial file has the problem too.  
Thanks for the line, I corrected.